skhep/math/vectors.py

```diff
@@ -104,22 +104,42 @@ def fromiterable(cls, values):
                 raise ValueError( 'Component #{0} is not a number!'.format(i) )
         return cls(values[0], values[1], values[2])
 
+    @property
     def x(self):
         """Return the x, aka first coordinate at position 0."""
         return self.__values[0]
 
+    @x.setter
+    def x(self, value):
+        """Sets x, aka first coordinate at position 0."""
+        self.__values[0] = value
+
+    @property
     def y(self):
-        """Return the x, aka second coordinate at position 1."""
+        """Return the y, aka first coordinate at position 1."""
```

Comment:
Enjoy the hols! I can edit things on the fly myself. Just wanted to make sure there was not something subtle being missed.